config/migrations/2024/bestuurseenheden/20240919101332-add-new-welzijnsvereniging-sint-anna/20240919101332-add-new-organisation-sint-anna.sparql

```diff
@@ -0,0 +1,54 @@
+PREFIX rdf:   <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX skos:  <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd:   <http://www.w3.org/2001/XMLSchema#>
+PREFIX dct:   <http://purl.org/dc/terms/>
+
+INSERT DATA {
+  GRAPH <http://mu.semte.ch/graphs/public> {
+
+      # Insert bestuurseenheid
+      <http://data.lblod.info/id/bestuurseenheden/66DB132B6ABCA6E6BF2DBBED> rdf:type	<http://data.vlaanderen.be/ns/besluit#Bestuurseenheid>, ;
+        <http://mu.semte.ch/vocabularies/core/uuid>	"66DB132B6ABCA6E6BF2DBBED" ;
+        skos:prefLabel	"Sint-Anna" ;
+        <http://www.w3.org/ns/adms#identifier>	"1011411080" ;
+        <http://mu.semte.ch/vocabularies/ext/kbonummer> "1011411080" ;
+        <http://www.w3.org/ns/regorg#orgStatus>	<http://lblod.data.gift/concepts/63cc561de9188d64ba5840a42ae8f0d6> ;
+        <http://data.vlaanderen.be/ns/besluit#classificatie>	<http://data.vlaanderen.be/id/concept/BestuurseenheidClassificatieCode/e8294b73-87c9-4fa2-9441-1937350763c9> .
+
+      # Bestuursorganen voor Sint-Anna
+      <http://data.lblod.info/id/bestuursorganen/40d628b0-6c5d-11ef-8f29-2728b827bd59> <http://data.vlaanderen.be/ns/besluit#bestuurt>	<http://data.lblod.info/id/bestuurseenheden/66DB132B6ABCA6E6BF2DBBED> .
+      <http://data.lblod.info/id/bestuursorganen/40bfba80-6c5d-11ef-8f29-2728b827bd59> <http://data.vlaanderen.be/ns/besluit#bestuurt>	<http://data.lblod.info/id/bestuurseenheden/66DB132B6ABCA6E6BF2DBBED> .
+      <http://data.lblod.info/id/bestuursorganen/40cab700-6c5d-11ef-8f29-2728b827bd59> <http://data.vlaanderen.be/ns/besluit#bestuurt>	<http://data.lblod.info/id/bestuurseenheden/66DB132B6ABCA6E6BF2DBBED> .
+
+      # Insert bestuursorganen
+      <http://data.lblod.info/id/bestuursorganen/40d628b0-6c5d-11ef-8f29-2728b827bd59> rdf:type	<http://data.vlaanderen.be/ns/besluit#Bestuursorgaan> ;
+        <http://mu.semte.ch/vocabularies/core/uuid>	"40d628b0-6c5d-11ef-8f29-2728b827bd59" ;
+        <http://data.vlaanderen.be/ns/besluit#classificatie>	<http://data.vlaanderen.be/id/concept/BestuursorgaanClassificatieCode/41caf7e6-b040-4720-9cc2-a96cfffed5b4> ;
+        <http://data.vlaanderen.be/ns/besluit#bestuurt>	<http://data.lblod.info/id/bestuurseenheden/66DB132B6ABCA6E6BF2DBBED> .
```

Comment:
This triple is already present in the `Bestuursorganen voor Sint-Anna` section.

Reply:
Left in for clarity/readability. Duplicate triples have no side effects.